wire-tests/MessageWithManyFields

```diff
@@ -0,0 +1,6300 @@
+import com.squareup.wire.FieldEncoding;
+import com.squareup.wire.ProtoAdapter;
+import com.squareup.wire.ProtoReader;
+import com.squareup.wire.ProtoWriter;
+import com.squareup.wire.WireField;
+import com.squareup.wire.internal.Internal;
+import java.io.IOException;
+import java.lang.Integer;
+import java.lang.Object;
+import java.lang.Override;
+import java.lang.String;
+import java.lang.StringBuilder;
+import java.util.List;
+import okio.ByteString;
+
+public final class Message extends com.squareup.wire.Message<Message, Message.Builder> {
```

Comment:
I think it’s going to be annoying to have this checked in. Drop it?

Reply:
We already have a mechanism for golden files which have messages with more than 16 messages and which need updated via the script in the root of the repo. This should sufficiently test the functionality of this PR without any added unit tests.

Reply:
I see, yes it tells why the tests are failing, so I am going to nuke this file and change one of those files